<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoinRoom</class>
 <widget class="QDialog" name="JoinRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {border-image: url(:/Images/UI/Keys.png);}</string>
  </property>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Karate</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font-family: 'Karate';
	border-radius: 10px;
	color: rgb(94, 44, 17);
}

QPushButton:hover {
	background-color: rgba(102, 32, 0, 0.3);
}
</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Enter 6 digit number</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>331</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(118, 79, 31, 150);
border-radius: 10px;</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="maxLength">
    <number>6</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="askUsernameLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Room Number To Join</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="enterButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Karate</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font-family: 'Karate';
	color: rgb(94, 44, 17);
	border-radius: 10px;
}

QPushButton:hover {
	background-color: rgba(102, 32, 0, 0.3);
}
</string>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
